Source/WebInspectorUI/UserInterface/Views/SpreadsheetCSSStyleDeclarationSection.js

```diff
@@ -318,6 +310,47 @@ WI.SpreadsheetCSSStyleDeclarationSection = class SpreadsheetCSSStyleDeclarationS
         this._renderSelector();
     }
 
+    _layoutGroupings()
+    {
+        if (!this._style.groupings.length) {
+            this._groupingsContainerElement?.remove();
+            this._groupingElements = [];
+            return;
+        }
+
+        if (!this._groupingsContainerElement) {
+            this._groupingsContainerElement = document.createElement("div");
+            this._groupingsContainerElement.classList.add("header-groupings");
+        } else
+            this._groupingsContainerElement.removeChildren();
+
+        let groupings = this._style.groupings.reverse();
```

Comment:
this is an in-place modification, so it will flip the order each time it's called

i think we either need to `.slice().reverse()` or just do a `for` that goes from `.length - 1` to `0`